automation/Extension/firefox/data/create_content_policy_table.sql

```diff
@@ -6,5 +6,5 @@ CREATE TABLE IF NOT EXISTS content_policy(
 	request_origin TEXT,
 	mime_type_guess TEXT,
 	page_id INTEGER,
-        top_url TEXT
+        visit_id INTEGER,
```

Comment:
Let's fix the indentation on the extension sql files while we're making changes here.